src/opcodes.rs

```diff
@@ -1168,11 +1219,26 @@ lazy_static! {
             .with_rmw()
             .with_rm_regw()
             .into_table(&mut ops);
+        //0x0F B6 /r MOVZX rW,rm8
+        define_opcode(0xB6).is_two_byte_op().calls(movzx_8bit).with_gas(Low)
+            .with_rm_regw()
+            .with_rm8()
+            .into_table(&mut ops);
         //0x0F B7 /r    MOVZX r32,rm16
         define_opcode(0xB7).is_two_byte_op().calls(movzx_16bit).with_gas(Low)
             .with_rm_reg32()
             .with_rm16()
             .into_table(&mut ops); 
+        //0x0F B6 /r MOVSX rW,rm8
+        define_opcode(0xBE).is_two_byte_op().calls(movsx_8bit).with_gas(Low)
```

Comment:
0x0F B6 is MOVZX, not MOVSX

Reply:
Er, actually just that the comment doesn't match the actual opcode used

Reply:
will fix